app/src/main/res/layout/activity_levels_view.xml

```diff
@@ -6,179 +6,209 @@
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.400"
-        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="@id/barrier2"
         app:layout_constraintTop_toBottomOf="@+id/fifthLevel"
-        app:layout_constraintVertical_bias="0.020" />
-
+        app:layout_constraintVertical_bias="0.020"
+        tools:ignore="TextViewEdits" />
 
     <TextView
         android:id="@+id/seventhLevel"
-        android:layout_width="630dp"
-        android:layout_height="40dp"
-        android:ems="10"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
         android:fontFamily="sans-serif-black"
         android:inputType="text"
         android:textAlignment="viewStart"
-        android:textSize="19sp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="14sp"
+        app:autoSizeMaxTextSize="24sp"
+        app:autoSizeStepGranularity="1sp"
+        app:layout_constraintBottom_toTopOf="@+id/eighthLevel"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.400"
-        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="@id/barrier2"
         app:layout_constraintTop_toBottomOf="@+id/sixthLevel"
-        app:layout_constraintVertical_bias="0.041" />
+        app:layout_constraintVertical_bias="0.020"
+        tools:ignore="TextViewEdits" />
 
     <TextView
         android:id="@+id/eighthLevel"
-        android:layout_width="630dp"
-        android:layout_height="40dp"
-        android:ems="10"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
         android:fontFamily="sans-serif-black"
         android:inputType="text"
         android:textAlignment="viewStart"
-        android:textSize="19sp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="14sp"
+        app:autoSizeMaxTextSize="24sp"
+        app:autoSizeStepGranularity="1sp"
+        app:layout_constraintBottom_toTopOf="@+id/ninethLevel"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.400"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="@+id/seventhLevel"
-        app:layout_constraintVertical_bias="0.167" />
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="@id/barrier2"
+        app:layout_constraintTop_toBottomOf="@+id/seventhLevel"
+        app:layout_constraintVertical_bias="0.020"
+        tools:ignore="TextViewEdits" />
 
     <TextView
         android:id="@+id/ninethLevel"
```

Comment:
Please keep track of IDEs suggestions :D There is a typo
<img width="447" alt="Screenshot 2022-08-11 at 10 37 58" src="https://user-images.githubusercontent.com/19372220/184095552-667fed44-8471-4fc6-aaea-fa1467831d3c.png">